### DISCUSSION TITLE
Trigger volume element

### DISCUSSION THREAD

**Discussion by Supriya-rathod** [mozilla/hubs#5409] [Art Questions] [answered]
hi friends,
as i was exploring building scene in editor , gone through assets element where i was exploring all the elements over i couldn't get one element which is "TRIGGER VOLUME" element , what it is? and what it is used for? can somebody help me with this it would be pleasure.
![WhatsApp Image 2022-05-04 at 4 55 16 PM](https://user-images.githubusercontent.com/47414593/166672805-aba2232c-8dec-4934-837a-bd6e416a19ea.jpeg)

**Comment by matthewbcool** [comment, marked as answer]
Trigger Volume is an **experimental component**.  There are currently several known issues and bugs and it relies on the developer community to fix/improve.

A trigger volume can be configured to start or pause a looped animation on another object in the scene.

It is not networked, so pausing/starting an animation will only be viewed by the local client.

Here is a great example scene from Sage Freeman: https://hubs.mozilla.com/scenes/Y5iwzvo
Also note there is no 'collision' on the gate as movement is entirely determined by the [nav-mesh](https://hubs.mozilla.com/labs/what-is-a-nav-mesh/).

**Reply by Supriya-rathod** [reply]
thankyou.

**Reply by Supriya-rathod** [reply]
hi, 
i was trying to use trigger volume referring your given example, my trigger volume element is not working when applying on target object and added loop-animation to both object and trigger volume element in component and with  property as paused its not working, am i correct with these step or wrong. if wrong can you please guide me with steps.

**Reply by matthewbcool** [reply]
Try searching discord for more implementation details and specific questions or create a new thread in discussions. https://discord.gg/sBMqSjCndj
